BuildAndTest.cmd

```diff
@@ -81,6 +81,15 @@ goto ExitFailed
 @REM Run all tests
 SET PASSED=true
 
+mstest /testContainer:bld\bin\Sarif.Converters.UnitTests\AnyCPU_%Configuration%\Sarif.Converters.UnitTests.dll
```

Comment:
>mstest [](start = 0, length = 6)

We were never running these before?? Good catch.

Reply:
Yep. Neglected this change when we refactored the converters.

---
In reply to: [136628913](https://github.com/Microsoft/sarif-sdk/pull/614#discussion_r136628913) [](ancestors = 136628913)